### DISCUSSION TITLE
Are the new "near real time" performance metrics supported?

### DISCUSSION THREAD

**Discussion by dannyhw** [invertase/react-native-firebase#5474] [Q&A] [answered]
It seems like recently there was an update that enables near real time performance metrics in firebase, I'm wondering if the current react-native-firebase package supports this new "near real time" type of metric. The firebase console warns me to make sure I use "SDK version 19.0.10 or newer" for android and 7.3 for ios.

I've recently updated my react-native-firebase packages but I still see this warning. I've been looking around the docs and I can't find any mention of it. 

So are the near real time metrics supported or are there plans to support it?

**Comment by mikehardy** [comment, marked as answer]
To know what underlying SDKs are in use examine the package.json in `@react-native-firebase/app` in order to see what underlying SDKs are in use:

https://github.com/invertase/react-native-firebase/blob/daaa72d82df6b9e5e2c1247c10792d4b12683541/packages/app/package.json#L63-L75

or check the CHANGELOG where we note them https://github.com/invertase/react-native-firebase/blob/master/CHANGELOG.md#features

unless you have overridden them https://rnfirebase.io/#overriding-native-sdk-versions

**Reply by dannyhw** [reply]
Ok thanks! That helps a lot 🙏.
